Makefile

```diff
@@ -51,6 +51,7 @@ build_nodejs::
 		yarn install && \
 		yarn run tsc --version && \
 		yarn run tsc && \
+		yarn test && \
```

Comment:
This should be in the provider build not the sdk build

Reply:
ah you are correct! fixing now